services/tenant-ui/frontend/src/components/profile/JWT.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <h2>{{ $t('profile.token') }}</h2>
+  <Accordion :multiple="true">
+    <AccordionTab header="Encoded JWT">
+      <div style="display: flex">
+        <div style="overflow: auto; width: 100%">
+          <!-- Just used p-inputtext to keep styling consistent -->
+          <div class="p-inputtext" style="overflow: auto">
```

Comment:
Should stay away from inline styles as much as we can. App uses https://www.primefaces.org/primeflex/ which provides framework classes for these if needed 

https://www.primefaces.org/primeflex/flex
https://www.primefaces.org/primeflex/overflow
https://www.primefaces.org/primeflex/width

(or apply with a named class in the style block of the component or something).